dashboard/db/migrate/20160225192540_add_serialized_attributes_and_type_to_tasks.rb

```diff
@@ -0,0 +1,8 @@
+class AddSerializedAttributesAndTypeToTasks < ActiveRecord::Migration
+  def change
+    add_column :plc_tasks, :type, :string, null: false, default: 'Plc::Task'
+    add_column :plc_tasks, :properties, :text
+
+    Plc::Task.update_all(properties: '{}')
```

Comment:
Actually it doesn't break - I think I was breaking something else earlier. I can remove this line.